src/jmcomic_ai/cli.py

```
@@ -160,10 +183,10 @@ def install_skills(
 
 @skills_app.command("uninstall")
 def uninstall_skills(
-    target_dir: Path | None = typer.Argument(
-        None, help="Directory to uninstall skills from. Defaults to ~/.claude/skills/jmcomic"
-    ),
-    yes: bool = typer.Option(False, "--yes", "-y", help="Skip confirmation prompt"),
+        target_dir: Path | None = typer.Argument(
+            None, help="Directory to uninstall skills from. Defaults to ~/.claude/skills/jmcomic"
+        ),
```

Ruff (0.14.11)

Do not perform function call `typer.Argument` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)